src/System.AspNetCore/CallContextManager.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Threading;
+using Microsoft.Omex.System.Context;
+using Microsoft.Omex.System.TimedScopes;
```

Comment:
nit: order alphabetically. Adam is on holidays so somebody has to point this out :).